src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall implements Cloneable {
```

Comment:
Why did you decide to implement Cloneable? We do not need it.
For all models.

Reply:
unfortunately, this was not a moderate decision, I was copying other models.